app/integrations/server/lib/updateHistoryWebhook.ts

```diff
@@ -0,0 +1,134 @@
+import { Random } from 'meteor/random';
+import _ from 'underscore';
+
+import * as Models from '../../../models/server';
+
+interface IKeyable {
+	type: string;
+	[key: string]: any;
+}
+
+class UpdateHistoryWebhook {
+	updateHistory({
+		historyId,
+		step,
+		integration,
+		event,
+		data,
+		triggerWord,
+		ranPrepareScript,
+		prepareSentMessage,
+		processSentMessage,
+		resultMessage,
+		finished,
+		url,
+		httpCallData,
+		httpError,
+		httpResult,
+		error,
+		errorStack,
+	}:
+	{
+		historyId: any;
```

Comment:
Can't we use something different than `any` here? I wonder if we can get the types these props will hold, so we don't have to type them as any always.

Reply:
I definitely agree, I would appreciate some help in trying to identify the types however, as the original code is in js and there are few clues to what type the parameter is supposed to be

Reply:
I think that you can log the input values and infer a type from there 👀

Reply:
Ok, I'll start working on it